sasmodels/__pyinstaller/hook-sasmodels.py

```diff
@@ -18,7 +18,14 @@
 
     hiddenimports = [
         "pyopencl",
+        "sasmodels.compare_many",
+        "sasmodels.guyou",
+        "sasmodels.jitter",
+        "sasmodels.list_pars",
+        "sasmodels.multiscat",
+        "sasmodels.special",
```

Comment:
Keeping this list update to date will be tedious and error prone. Can we generate a list of modules by scanning the directory?

Reply:
Yes, as a future enhancement, it looks like the following works:

```python
from PyInstaller.utils.hooks import collect_submodules

...

hiddenimports = collect_submodules("sasmodels") + [
    "pyopencl",
]
```

That seems to pick up all of the files that have been added into the above.

Explanation: `collect_submodules` uses `pkgutil.iter_modules` which finds all modules that exist on disk; by contrast, my understanding is that the pyinstaller `Analyze` step is only looking for the use of `import` which misses the files/modules that are never imported.